src/core/agent/services/identifierService.ts

```diff
@@ -303,6 +311,106 @@ class IdentifierService extends AgentService {
       );
     }
   }
+
+  async createMultisig(
+    identifier: string,
+    contacts: KeriContact[],
+    meta: Pick<
+      IdentifierMetadataRecordProps,
+      "displayName" | "colors" | "theme"
+    >
+  ): Promise<string | undefined> {
+    const metadata = await this.getMetadataById(identifier);
+    this.validIdentifierMetadata(metadata);
+    const aid = (await this.agent.modules.signify.getIdentifierByName(
+      metadata.signifyName as string
+    )) as Aid;
+    const otherAids = [];
+    for (const contact of contacts) {
+      const aid = await this.agent.modules.signify.resolveOobi(contact.oobi);
+      otherAids.push({
+        state: aid.response,
+      } as Pick<Aid, "state">);
+    }
+    const result = await this.agent.modules.signify.createMultisig(
+      aid,
+      otherAids
+    );
+    return this.createIdentifier({
+      method: IdentifierType.KERI,
+      displayName: meta.displayName,
+      colors: meta.colors,
+      theme: meta.theme,
+      signifyName: result.name,
+      opName: result.op.name, //we save the opName here to sync the multisig's status later
+      isPending: result.op.done ? false : true, //this will be updated once the operation is done
+    });
+  }
+  async isJoinedCreateMultisig(msgSaid: string): Promise<boolean> {
```

Comment:
So I presume `createMultisig` is if we initiate it, but `joinCreateMultisig` is used if another initiates the multi-sig request and we accept it?

When is `isJoinedCreateMultisig` used? To me this seems to check if we have an identifier by that `signifyName` from the msg, but does this mean it's complete or no?

Reply:
- Yes. The `createMultisig` is used to initiate a multisig and the `joinCreateMultisig` is used to accept the request that is created by other people (I will rename it to `joinMultisig`). 
- `isJoinedCreateMultisig` is used to check if a user has already joined the multisig. It can be a pending multisig. We have a key called `isPending` in our metadata so we can check if it is done or not.

Reply:
OK, so `isJoinedCreateMultisig` is just used to check if we have joined a multi-sig, pending or not pending? It's not used yet in the current PR so I'm guessing you still need to commit some code for me to check

Reply:
It will be called inside the joinMultisig to avoid joining the same multisig.